src/init.cpp

```diff
@@ -1515,6 +1527,13 @@ bool AppInitParameterInteraction()
         }
     }
 
+    fDisableGovernance = gArgs.GetBoolArg("-disablegovernance", false);
+    LogPrintf("fDisableGovernance %d\n", fDisableGovernance);
+
+    if (fDisableGovernance) {
+        InitWarning(_("You are starting with governance validation disabled. This is expected if you are running a pruned node"));
```

Comment:
We could do something like this at least? Just to not show any stuff about pruning if someone only disables governance without touching `-prune`.  If not, there is a `.` missing at the end. 

```suggestion
        InitWarning(strprintf(_("You are starting with governance validation disabled. %s"), fPruneMode ? _("This is expected because you are running a pruned node.") : ""));
```

Reply:
Good idea IMO but should probably be more like
```suggestion
        InitWarning(_("You are starting with governance validation disabled.") + (fPruneMode ? " " + _("This is expected because you are running a pruned node.") : ""));
```